tools/lib/provision.py

```diff
@@ -179,7 +179,11 @@ def main(options):
 
     # setup-apt-repo does an `apt-get update`
     run(["sudo", "./scripts/lib/setup-apt-repo"])
```

Comment:
I think for these, I'd rather just do the overall `provision` retry than adding lines to every place we do a network thing in provision; there's a lot of them!